app/src/main/java/ua/in/badparking/ui/activities/MainActivity.java

```diff
@@ -47,7 +47,6 @@
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
 
-        setContentView(R.layout.activity_main);
```

Comment:
yeah, already saw that, sorry. Your PR is merged

On 7/12/16, Alex Belchenko <notifications@github.com> wrote:
>> @@ -47,7 +47,6 @@
>>      protected void onCreate(Bundle savedInstanceState) {
>>          super.onCreate(savedInstanceState);
>>
>> -        setContentView(R.layout.activity_main);
>
> Because this activity is already annotated with @ContentView annotation,
> that doing actually the same
>
> ---
> You are receiving this because you modified the open/close state.
> Reply to this email directly or view it on GitHub:
> https://github.com/badparking/badparking-android/pull/29/files/eaade6c23a01c9b9711f4e2a18dcc21126c7ec8b#r70386782